docs/api-guide/parsers.md

```diff
@@ -87,7 +87,7 @@ You will typically want to use both `FormParser` and `MultiPartParser` together
 
 ## MultiPartParser
 
-Parses multipart HTML form content, which supports file uploads.  Both `request.data` will be populated with a `QueryDict`.
+Parses multipart HTML form content, which supports file uploads. `request.data` will be populated with a `QueryDict`.
```

Comment:
I think that "both `request.data` and `request.files` ..." is more accurate.

Reply:
Thanks for responding.

I decided to test whether our modifications are accurate and came across these issues.

- `request.files` doesn't exist
- `request.FILES` exists.
```python
request.files
*** AttributeError: 'Request' object has no attribute 'files'

request.FILES
<MultiValueDict: {'image': [<TemporaryUploadedFile: ____.jpg (image/jpeg)>]}>

```

I am hesitant about using `request.FILES`  [because I came across this deprecation notice](https://www.django-rest-framework.org/community/3.1-announcement/#deprecations).